assets/js/modules/reader-revenue-manager/components/dashboard/ReaderRevenueManagerSetupCTABanner.js

```diff
@@ -37,268 +35,164 @@ import { __ } from '@wordpress/i18n';
-import whenInactive from '../../../../util/when-inactive';
-import { withWidgetComponentProps } from '../../../../googlesitekit/widgets/util';
-import { CORE_MODULES } from '../../../../googlesitekit/modules/datastore/constants';
 import { CORE_USER } from '../../../../googlesitekit/datastore/user/constants';
-import {
-	READER_REVENUE_MANAGER_MODULE_SLUG,
-	READER_REVENUE_MANAGER_SETUP_BANNER_DISMISSED_KEY,
-} from '../../datastore/constants';
-import { Cell, Grid, Row } from '../../../../material-components';
+import { READER_REVENUE_MANAGER_MODULE_SLUG } from '../../datastore/constants';
 import SetupSVG from '../../../../../svg/graphics/reader-revenue-manager-setup.svg';
 import SetupTabletSVG from '../../../../../svg/graphics/reader-revenue-manager-setup-tablet.svg';
 import SetupMobileSVG from '../../../../../svg/graphics/reader-revenue-manager-setup-mobile.svg';
-import Link from '../../../../components/Link';
-import { trackEvent, WEEK_IN_SECONDS } from '../../../../util';
 import {
 	AdminMenuTooltip,
 	useShowTooltip,
 	useTooltipState,
 } from '../../../../components/AdminMenuTooltip';
-import useViewContext from '../../../../hooks/useViewContext';
-
-function ReaderRevenueManagerSetupCTABanner( { Widget, WidgetNull } ) {
-	const viewContext = useViewContext();
+import { CORE_NOTIFICATIONS } from '../../../../googlesitekit/notifications/datastore/constants';
+import NotificationWithSVG from '../../../../googlesitekit/notifications/components/layout/NotificationWithSVG';
+import LearnMoreLink from '../../../../googlesitekit/notifications/components/common/LearnMoreLink';
+import ActionsCTALinkDismiss from '../../../../googlesitekit/notifications/components/common/ActionsCTALinkDismiss';
+import { WEEK_IN_SECONDS } from '../../../../util';
+
+export default function ReaderRevenueManagerSetupCTABanner( {
+	id,
+	Notification,
+} ) {
 	const breakpoint = useBreakpoint();
-	const isMobileBreakpoint = breakpoint === BREAKPOINT_SMALL;
-	const isTabletBreakpoint = breakpoint === BREAKPOINT_TABLET;
 
 	const onSetupActivate = useActivateModuleCallback(
 		READER_REVENUE_MANAGER_MODULE_SLUG
 	);
 
-	const onSetupCallback = useCallback( () => {
-		trackEvent(
-			`${ viewContext }_rrm-setup-notification`,
-			'confirm_notification'
-		).finally( () => {
-			onSetupActivate();
-		} );
-	}, [ onSetupActivate, viewContext ] );
+	const showTooltip = useShowTooltip( id );
+	const { isTooltipVisible } = useTooltipState( id );
 
-	const showTooltip = useShowTooltip(
-		READER_REVENUE_MANAGER_SETUP_BANNER_DISMISSED_KEY
-	);
-	const { isTooltipVisible } = useTooltipState(
-		READER_REVENUE_MANAGER_SETUP_BANNER_DISMISSED_KEY
-	);
+	const { triggerSurvey } = useDispatch( CORE_USER );
 
-	const isDismissed = useSelect( ( select ) =>
-		select( CORE_USER ).isPromptDismissed(
-			READER_REVENUE_MANAGER_SETUP_BANNER_DISMISSED_KEY
-		)
-	);
+	const readerRevenueManagerDocumentationURL =
+		'https://readerrevenue.withgoogle.com';
```

Comment:
Is the variable necessary? I only see it used once. If not, let's inline it below.